recommendation/system.web.mvc.json

```diff
@@ -269,14 +303,427 @@
               "Type": "Using",
               "Value": "Microsoft.AspNetCore.Mvc.Filters",
               "Description": "Add Microsoft.AspNetCore.Mvc.Filters namespace.",
-              "ActionValidation":{"Contains":"using Microsoft.AspNetCore.Mvc.Filters;", "NotContains":""}
+              "ActionValidation": {
+                "Contains": "using Microsoft.AspNetCore.Mvc.Filters;",
+                "NotContains": ""
+              }
             },
             {
               "Name": "RemoveDirective",
               "Type": "Using",
               "Value": "System.Web.Mvc",
               "Description": "Remove System.Web.Mvc namespace.",
-              "ActionValidation":{"Contains":"", "NotContains":"using System.Web.Mvc;"}
+              "ActionValidation": {
+                "Contains": "",
+                "NotContains": "using System.Web.Mvc;"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "Type": "Method",
+      "Name": "System.Web.Mvc.Controller.HttpNotFound()",
+      "Value": "System.Web.Mvc.Controller.HttpNotFound()",
+      "KeyType": "Name",
+      "ContainingType": "Controller",
+      "RecommendedActions": [
+        {
+          "Source": "Amazon",
+          "Preferred": "Yes",
+          "TargetFrameworks": [
+            {
+              "Name": "netcoreapp3.1",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            },
+            {
+              "Name": "net5.0",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            }
+          ],
+          "Description": "Replace HttpNotFound with NotFound and replace System.Web.Mvc namespace with Microsoft.AspNetCore.Mvc.",
+          "Actions": [
+            {
+              "Name": "ReplaceMethodOnly",
+              "Type": "Method",
+              "Value": {
+                "oldMethod": "HttpNotFound",
+                "newMethod": "NotFound"
+              },
+              "Description": "Replace HttpNotFound with NotFound",
+              "ActionValidation": {
+                "NotContains": "HttpNotFound"
```

Comment:
Should we have a contains here just for consistency?

Reply:
Sure, updated.